packages/gatsby/cache-dir/head/utils.js

```diff
@@ -40,14 +40,14 @@ if (process.env.NODE_ENV !== `production`) {
 }
 
 /**
- * Warn for invalid tags in head.
+ * Warn for invalid tags in Head which may have been directly added or introduced by `wrapRootElement`
  * @param {string} tagName
  */
 export function warnForInvalidTags(tagName) {
   if (process.env.NODE_ENV !== `production`) {
-    const warning = `<${tagName}> is not a valid head element. Please use one of the following: ${VALID_NODE_NAMES.join(
+    const warning = `<${tagName}> is not a valid head element. You can only use one of the following: ${VALID_NODE_NAMES.join(
```

Comment:
```suggestion
    const warning = `<${tagName}> is not a valid head element. Please use one of the following: ${VALID_NODE_NAMES.join(
```

I like the old phrasing better, it's warmer

Reply:
👍